lib/main.dart

```diff
@@ -114,47 +74,90 @@ class _MyHomePageState extends State<MyHomePage> {
   ];
 
   Future<void> _showChoiceDialog(BuildContext context) {
-    return showDialog(context: context, builder: (BuildContext context) {
-      return AlertDialog(
-        title: const Center(child: Text('Make a Choice...')),
-        content: SingleChildScrollView(
-          child: Row(
-            mainAxisAlignment: MainAxisAlignment.spaceAround,
-            children: <Widget>[
-              ElevatedButton(
-                  onPressed: () => pickImageFromGallery(),
+    return showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return AlertDialog(
+          title: const Center(
+            child: Text('Make a Choice...'),
+          ),
+          content: SingleChildScrollView(
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: <Widget>[
```

Comment:
When you have array of objects of the same class, in this case it is `Widget`, you can not to show array type exclusively, but just an array which contains widgets - `[Widget1(), Widget2() ..etc]`

Reply:
fixed successfully